assets/controller.yaml

```diff
@@ -107,6 +107,11 @@ spec:
           env:
             - name: ADDRESS
               value: /var/lib/csi/sockets/pluginproxy/csi.sock
+            - name: ENABLE_TOPOLOGY
+              valueFrom:
+                configMapKeyRef:
+                  name: cloud-conf
+                  key: enable_topology
```

Comment:
I want to watch CI to see if this is populated at the right time. As I understand it, the pod will fail to deploy under the secret is created, which will happen the first time the controller runs. That's probably what we want, rather than providing a default and having to watch for changes to this.